Makefile

```diff
@@ -186,14 +186,25 @@ sqs-load-test-ui:
 sqs-profile:
 	go tool pprof -http=:8080 http://localhost:9092/debug/pprof/profile?seconds=15
 
+# Validates that SQS concentrated liquidity pool state is
+# consistent with the state of the chain.
+sqs-validate-cl-state:
+	ingest/sqs/scripts/validate-cl-state.sh "http://localhost:9092"
+
+# Compares the quotes betwen SQS and chain over pool 1136
+# which is concentrated.
+sqs-quote-compare:
+	ingest/sqs/scripts/quote.sh "http://localhost:9092"
+
```

Comment:
should all of these hardcode 9092, or pull the port from the config file instead?

Reply:
Since 9092 is default SQS and these are test scripts to help with debugging, I think we can keep hardcoded. I would agree with a config for prod deployments configurations or similar